internal/service/rds/cluster_blue_green_resource.go

```diff
@@ -0,0 +1,1482 @@
+										}
+										return "[DEBUG] Error:", err
+									},
+
+									func(err error) (bool, error) {
+										// Retry for IAM eventual consistency.
+										if tfawserr_sdkv2.ErrMessageContains(err, errCodeInvalidParameterValue, "IAM role ARN value is invalid or does not include the required permissions") {
+											return true, err
+										}
+
+										if tfawserr_sdkv2.ErrMessageContains(err, errCodeInvalidParameterCombination, "disable deletion pro") {
+											return true, err
+										}
+
+										return false, err
+									},
+								)
+
+								cleaupWaiters = append(cleaupWaiters, func(optFns ...tfresource.OptionsFunc) {
+									_, err = waitDBInstanceDeleted(ctx, meta.(*conns.AWSClient).RDSConn(ctx), aws.StringValue(instance.DBInstanceIdentifier), deadline.Remaining(), optFns...)
+									if err != nil {
+										diags = sdkdiag.AppendErrorf(diags, "updating RDS DB Instance (%s): deleting Blue/Green Deployment source: waiting for completion: %s", instance, err)
+									}
+								})
+							}
+
+							deleteInput := &rds.DeleteDBClusterInput{
+								DBClusterIdentifier: aws.String(d.Get("cluster_identifier").(string)),
+							}
+			
+							clusters, _ := conn.DescribeDBClusters(descClusters)
+			
+							_, _ = tfresource.RetryWhen(ctx, 5*time.Minute,
+								func() (any, error) {
+									return conn.DeleteDBCluster(deleteInput)
+								},
+								func(err error) (bool, error) {
+									// Retry for IAM eventual consistency.
+									if tfawserr_sdkv2.ErrMessageContains(err, errCodeInvalidParameterValue, "IAM role ARN value is invalid or does not include the required permissions") {
+										return true, err
+									}
+			
+									if tfawserr_sdkv2.ErrMessageContains(err, errCodeInvalidParameterCombination, "disable deletion pro") {
+										return true, err
+									}
+			
+									cleaupWaiters = append(cleaupWaiters, func(optFns ...tfresource.OptionsFunc) {
+										_, err = waitBlueGreenClusterDeploymentDeleted(ctx, connv2, aws.StringValue(&bluegreen[0]), deadline.Remaining(), optFns...)
+										if err != nil {
+											diags = sdkdiag.AppendErrorf(diags, "updating RDS DB Cluster (%s): deleting Blue/Green Deployment: waiting for completion: %s", d.Id(), err)
+										}
+									})
+								},
+							)
+
+							bginput := &rds_sdkv2.DeleteBlueGreenDeploymentInput{
+								BlueGreenDeploymentIdentifier: aws.String(bluegreen[0]),
+								DeleteTarget:                  aws.Bool(false),
+							}
+
+							if err != nil {
+								log.Printf("[DEBUG] updating RDS DB Cluster (%s): deleting Blue/Green Deployment source: %s", aws.StringValue(&bluegreen[0]), err)
+							}
+
+							_, err = connv2.DeleteBlueGreenDeployment(ctx, bginput)
+
+							if err != nil {
+								log.Printf("[DEBUG] updating RDS DB Cluster (%s): deleting Blue/Green Deployment source: %s", aws.StringValue(&bluegreen[0]), err)
+							}
+
+						}
+						return true, err
+					},
```

Comment:
internal/service/rds/cluster_blue_green_resource.go:979:7: syntax error: unexpected comma at end of statement